src/Features/Core/Portable/DisposeAnalysis/DisposeObjectsBeforeLosingScopeDiagnosticAnalyzer.cs

```diff
@@ -125,13 +125,6 @@ protected override void InitializeWorker(AnalysisContext context)
                     ComputeDiagnostics(disposeDataAtExit, notDisposedDiagnostics, mayBeNotDisposedDiagnostics,
                         disposeAnalysisResult, pointsToAnalysisResult);
 
-                    if (disposeAnalysisResult.ControlFlowGraph.OriginalOperation.HasAnyOperationDescendant(o => o.Kind == OperationKind.None))
```

Comment:
Thanks @chsienki. On second thought, I am wondering if we should leave these checks in place to future proof the analyzers from false positives in case of future language features with unimplemented IOperation support.

Reply:
I did wonder that too. Perhaps we should Debug.Assert? Given that we shouldn't have none operations in reality, it will stop anything breaking, but save as a catch for future language features?

Reply:
Unfortunately, we do have bunch of nodes generated with [OperationKind.None](http://source.roslyn.io/#Microsoft.CodeAnalysis/Generated/OperationKind.Generated.cs,2af646b24dfbee5f,references) due to pending IOperation support (attributes, query clauses, unsafe code, VB xml literals, etc.)

Reply:
Ah ok. I guess we'll just leave this as-is then.